<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="Create">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>391</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>35</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>35</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <item row="0" column="0">
     <widget class="QLineEdit" name="first_name">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Förnamn</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="last_name">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="placeholderText">
       <string>Efternamn</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="username">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="placeholderText">
       <string>Användarnamn</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="email">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="placeholderText">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="password_user">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Lösenord</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="confirm_password">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
      </property>
      <property name="placeholderText">
       <string>Bekräfta Lösenord</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="create">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(44, 101, 164);
border-radius: 5px;
color: rgb(255, 255, 255);
height: 30px;</string>
      </property>
      <property name="text">
       <string>Skapa</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>400</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../Bilder/Skärmavbild 2024-05-02 kl. 00.15.18.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
